pkg/blob/fetch.go

```diff
@@ -118,14 +118,18 @@ func extractTar(reader io.Reader, dir string) error {
 				return err
 			}
 		case tar.TypeReg:
-			outFile, err := os.Create(filePath)
+			outFile, err := os.OpenFile(filePath, os.O_RDWR|os.O_CREATE|os.O_TRUNC, header.FileInfo().Mode())
 			if err != nil {
 				return err
 			}
-			defer outFile.Close()
```

Comment:
why no longer defer?

Reply:
The defer was in a for loop and was getting complaints from the IDE. I suppose the defer gets lost with each iteration, or something equally bad happens when you defer in a for loop. So I just manually close the file.

Reply:
I can live with that!